src/tools/native-client/pom.xml

```diff
@@ -151,6 +151,61 @@
                     <argLine>-Djava.library.path=_build/</argLine>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
would you publish natice codec to maven center ?

Reply:
Yeah, then we can use in nebula-java